userspace/libsinsp/cri.hpp

```diff
@@ -40,12 +39,12 @@ namespace libsinsp
 {
 namespace cri
 {
-std::vector<std::string> s_cri_unix_socket_paths;
-int64_t s_cri_timeout = 1000;
-int64_t s_cri_size_timeout = 10000;
-sinsp_container_type s_cri_runtime_type = CT_CRI;
-std::string s_cri_unix_socket_path;
-bool s_cri_extra_queries = true;
+inline std::vector<std::string> s_cri_unix_socket_paths;
```

Comment:
Care to explain why was this needed?

Reply:
This is a header file, as such it could be included in several compilation units. This is not the case today, so there is no problem, but if it were included again we'd have a linker error for duplicate symbols. Inline global variables are a way to tell the compiler/linker that only one instance of that should be created, regardless of the number of times it is encountered in different compilation units.
The other approach would be to declare it as external and define it only once in a separate .cpp file; we now have the inline option since we upgraded the C++ standard a while ago.